ibis_substrait/tests/compiler/test_decompiler.py

```diff
@@ -260,4 +260,9 @@ def test_searchedcase(compiler):
     expr = t.aggregate(t.ork.like("THING%").ifelse(t.bork32, 0).sum())
     plan = compiler.compile(expr)
     (result,) = decompile(plan)
-    assert result.equals(expr)
+    # Is this gross?  Yes, yes it is.
+    # gforsyth things this is because some underlying literal value
+    # in the reconstructed expression is getting cast in some funky
+    # way so that the expressions aren't _strictly_ (to ibis) equivalent
+    # even though they are.
+    assert repr(result) == repr(expr)
```

Comment:
This might be our old friend equality comparison of upcasted literals rearing its head.

Reply:
You know, I never liked him, anyway

Reply:
they do compare equal under ibis 3.x -- thoughts on skipping or xfailing for ibis 2.x?